README.md

```diff
@@ -59,24 +65,22 @@ Translations of the guide are available in the following languages:
 * [Portuguese]
 * [Spanish]
 
-### Source Code Layout
+## Formatting
 
-* <a name="spaces-indentation"></a>
-  Use two **spaces** per indentation level.
-  No hard tabs.
-  <sup>[[link](#spaces-indentation)]</sup>
+Elixir v1.6 introduced a [Code Formatter] and [Mix format] task.
+The formatter should be preferred for all new projects and source code.
+The following formatting rules are applied automatically, but are provided here
```

Comment:
Just a minor thing: I would think that this can easily be missed so maybe a more visible note would be to bold the text that says "The following formatting rules are applied automatically..."

Maybe overkill. What are your thoughts?